tensorflow/core/ops/array_ops.cc

```diff
@@ -709,7 +709,6 @@ tf.diag(diagonal) ==> [[1, 0, 0, 0]
                        [0, 0, 0, 4]]
 ```
 
-diagonal: Rank k tensor where k is at most 3.
```

Comment:
Perhaps leave this argument description, but change it to say where k is at least 1 ?

Reply:
Has changed in code, but seems this dialog doesn't fold.